doc/source/whatsnew/v1.3.0.rst

```diff
@@ -217,6 +217,9 @@ Other enhancements
 - :class:`RangeIndex` can now be constructed by passing a ``range`` object directly e.g. ``pd.RangeIndex(range(3))`` (:issue:`12067`)
 - :meth:`round` being enabled for the nullable integer and floating dtypes (:issue:`38844`)
 - :meth:`pandas.read_csv` and :meth:`pandas.read_json` expose the argument ``encoding_errors`` to control how encoding errors are handled (:issue:`39450`)
+- :meth:`.GroupBy.any` and :meth:`.GroupBy.all` use Kleene logic with nullable data types (:issue:`37506`)
```

Comment:
hmm is this how we other groupby refs? i am not sure this will render

Reply:
I see one other entry with this style, I think it's good here to avoid having the duplication of `DataFrameGroupBy.`, `SeriesGroupBy.` for both `any` and `all`. Building docs now to check rendering

Reply:
Looks good:
<img width="871" alt="Screen Shot 2021-04-13 at 1 08 05 PM" src="https://user-images.githubusercontent.com/37011898/114592907-74c4a980-9c59-11eb-89d7-63f65f08f8b0.png">
(links to https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.core.groupby.GroupBy.all.html)

Reply:
> hmm is this how we other groupby refs? i am not sure this will render

Yes, that's something @rhshadrach recently found out. The leading `.` is basically a wildcard search for a matching object, so `.GroupBy` will find `pandas.core.groupby.GroupBy` (since we only have a single `GroupBy` class in our API docs)

Reply:
I do wonder if this is something in the docs we could find a better solution to - knowing whether to look under GroupBy or DataFrameGroupBy etc. I personally don't find it hurts usability of the docs, and is only a slight annoyance when crafting the whatsnew, but I wouldn't be surprised if users were confused by it.

Reply:
> I do wonder if this is something in the docs we could find a better solution to - knowing whether to look under GroupBy or DataFrameGroupBy etc

Yeah, I always need to look at https://pandas.pydata.org/docs/dev/reference/groupby.html which one is available to link to ..

In general I think we should try to have most of them available just using `GroupBy` to make life easier. 
We actually do have some duplication as well. Eg for some methods we have both the GroupBy and DataFrameGroupBy version (eg https://pandas.pydata.org/docs/dev/reference/api/pandas.core.groupby.DataFrameGroupBy.all.html#pandas.core.groupby.DataFrameGroupBy.all and https://pandas.pydata.org/docs/dev/reference/api/pandas.core.groupby.GroupBy.all.html#pandas.core.groupby.GroupBy.all). I think for those we can only have the GroupBy version.